src/test/csi-qingcloud/README.md

```diff
@@ -28,6 +28,11 @@ helm delete csi-qingcloud --namespace kube-system
 
 The command removes all the Kubernetes components associated with the chart and deletes the release.
 
+## Upgrade Notes
+
+The qingcloud-csi chart uses snapshotter v4.0 by default starting from v1.2.9, v1 crd is required for snapshotter v4.0.0 and above, so user must install v1 crd, otherwise the snapshot operation will be stuck.
```

Comment:
suggested:

The qingcloud-csi chart uses [csi-snapshotter](https://github.com/kubernetes-csi/external-snapshotter) v4.0.0 by default starting from chart v1.2.9, v1 snapshot CRDs are required for csi-snapshotter v4.0.0 and above, therefore user must install v1 snapshot CRDs prior to the upgrade, otherwise the snapshot operation will fail.

Reply:
csi-snapshotter v4.0.0 supports both v1 and v1beta1 snapshot objects.
User can specify chart value `snapshotter.tag=v2.0.1` with helm to use csi-snapshotter v2.0.1 which does not require v1 snapshot CRDs to be installed.